interval/tools.go

```diff
@@ -2,9 +2,28 @@ package interval
 
 import (
 	"fmt"
+	"github.com/vertgenlab/gonomics/numbers"
 )
 
 // CoordsToString takes in a interval and returns a string in the format of chr:start-end
 func CoordsToString(i Interval) string {
 	return fmt.Sprintf("%s:%v-%v", i.GetChrom(), i.GetChromStart(), i.GetChromEnd())
 }
+
+func IntervalSize(i Interval) int {
+	return i.GetChromEnd() - i.GetChromStart()
+}
+
+func OverlapSize(a, b Interval) int {
```

Comment:
Are we making the assumption these chromosomes are from the same genome?

Reply:
Yes! Good point. Will add a note to usage